_docs/style/colors.md

```diff
@@ -10,82 +10,553 @@ status: complete
+includes the Helix color palette:
+
+- xxxx
+- xxxx
+- xxxxx
+- xxxxx
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a (UI) that
+includes the Helix color palette:
+
+- xxxx
+- xxxx
+- xxxxx
+- xxxxx
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Accessibility
+
+Consider the following accessibility Guidelines when you construct a (UI) that
+includes the Helix color palette:
+
+- xxxx
+- xxxx
+- xxxxx
+- xxxxx
+
+### Color combination Do's and Don'ts
+
+A list of do's and don'ts...
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Application frame colors
+
+Description and visual of application frame colors...
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Element with light fill or border"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/colors/xxxxx.png" width="402"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Status colors
+
+The Helix color palette includes an array of colors that you can use to communicate the status of an entity, object, or process..
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Criticality of information
+
+Text here
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Element with light fill or border"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/colors/xxxxx.png" width="402"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Health status
+
+These colors and labels communicate the well-being of an infrastructure object, for example, a load balancer or a server, or an entity, for example, an account or a user.
+
+Because they tested well with customers, the Helix team encourage you to use the provided labels, although we recognize that certain use cases might deviate from these labels.
+
```

Comment:
i forgot to mention that these are interactive states, and should be documented the same way the rest of the interactive states are documented....should these interactive state color documentation live here, or should they be added to the interactive states as a health status interactive states section?